content/docs/start/model-management/model-registry.md

```diff
@@ -0,0 +1,297 @@
+model lifecycle. That can be useful for example if you want to trigger certain
+model registry actions programmatically. You can learn more about the details of
+GTO in its [documentation](/docs/gto).
+
+</details>
+
+<details>
+
+#### 💡 Expand to see how registering models works under the hood
+
+Registering the model version as we just did using DVC Studio is equivalent to
+the following GTO command
+
+```
+gto register pool-segmentation [ref] --version v1.0.0
+```
+
+Here, `[ref]` is the git reference/hash we selected from the menu in Studio.
+
+For more details you can have a look at the
+[gto register command reference](doc/gto/command-reference/register).
+
+</details>
+
+## Assigning lifecycle stages
+
+We have a first version for our model and now it is a good time to assign a
+model lifecycle stage to it. You can create any number of lifecycle stages with
+any names you wish but in this example we will only create two stages called
+"dev" and "test".
+
+Stages are created whenever a model version is assigned to them. We will now
+assign our model to the "dev" stage as follows.
+
+![Assigning model stages](/img/mr-assign-model-stage.gif)
+
+<details id="under-the-hood-assigning-model-stages">
+
+#### 💡 Expand to see how assigning model stages works under the hood
+
+Assigning the "dev" stage to the model as we just did using DVC Studio is
+equivalent to the following GTO command
+
+```
+gto assign pool-segmentation --version v1.0.0 --stage dev
+```
+
+For more details you can have a look at the
+[gto assign command reference](doc/gto/command-reference/assign).
+
+</details>
+
+## Changing (and removing) stage assignments
+
+Let's say that we've decided to promote our model version 1.0.0 to production.
+We will assign the model version to the "prod" stage just like we did with the
+"dev" stage in the previous section. When we assing the model to a stage, it can
+automatically trigger actions in our CICD workflows, like deploying the model to
+a new environment (we will explore how this is done in the [Using and Deploying
+models] (doc/start/model-management/model-cicd) chapter).
+
+Let's say that after some additional experimentation, we've improved our model
+and we would like to register a new version and assign it to the "dev" stage
+instead of our current version 1.0.0. You can change some parameters We would do
+this just like before, registering the new commit as model version 1.1.0 and
+assigning it to the "dev" stage in Studio. This will also automatically replace
+the assignment onf version 1.0.0 to the "dev" stage by version 1.1.0. Since the
+steps we would take are the same as before, we omit this in our example
+repository but you can try it out yourself.
+
+The detailed view of our model in the registry should now match what we see
+[in our example](<(https://studio.iterative.ai/team/Iterative/models/b3P4bcYIrGYdzyjqzsf9Xw==/pool-segmentation/v0.1.0)>).
```

Comment:
I'm a bit confused by this. We use a different version in the demo (v0.1.0 instead of v1.0.0; why?) and the demo is in `dev` stage. Is it expected that the user has transitioned to the prod stage at this point? It's unclear to me.

Reply:
It also seems like we lost the step to deprecate `dev` somewhere?

Reply:
I changed the versioning to v1.0.0 since that is the default suggested in Studio. The demo will be updated in the [PR](https://github.com/iterative/example-repos-dev/pull/271) to match this. I also added the "prod" stage there. Admittedly, it would make a bit more sense to do this in the next chapter. The main reason to do it now is to have more stuff happen before the "Auditing model history" paragraph.

I think we deceded not to deprecate dev, but I guess we can add it...it will also make the git history slightly more interesting and it is probably good to show at least one `gto deprecate`-type action.